phpunit.xml.dist

```diff
@@ -10,11 +10,12 @@
   -->
 
 <phpunit xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-         xsi:noNamespaceSchemaLocation="http://schema.phpunit.de/5.2/phpunit.xsd"
+         xsi:noNamespaceSchemaLocation="http://schema.phpunit.de/6.0/phpunit.xsd"
          backupGlobals="false"
          bootstrap="vendor/autoload.php"
          colors="true"
          enforceTimeLimit="true"
+         beStrictAboutTestsThatDoNotTestAnything="false"
```

Comment:
Not yet, but in PHPUnit 6.0 you will get a lot of risky tests and exit code 1 when you have no explicit asserts in them. This maintains 5.x behaviour for now.

But since I did not update the constraint yet, it is not strictly necessary.